protos/log_files/log_files.proto

```diff
@@ -34,6 +42,14 @@ message DownloadLogFileResponse {
     ProgressData progress = 2; // Progress if result is progress
 }
 
+message DownloadLogFileRequest {
+    Entry entry = 1; // Entry of the log file to download.
+    string path = 2; // Path of where to download log file to.
+}
```

Comment:
But it already exists for `SubscribeDownloadLogFile`.

This is a bit confusing. We now have a finite/async download as well as a finite/sync.
Could we just have one finite/both? I guess that's not supported, right?

Reply:
The `Subscribe*` one represents a stream, which is equivalent to a function that takes a callback as a parameter and sends the result through it. Whereas the blocking call does not take any argument and returns the result.

So if really there is a need for the blocking function, it needs to be done this way.

Now one could argue that MAVSDK should not expose the blocking function here, because it's not that difficult for the user to use a future/promise in C++. In Python it's blocking (unless you run it in a separate coroutine), and in Java/Swift the API is made such that you can easily make it blocking.

So it's really just for helping C++ users not write a future/promise to block the `SubscribeDownloadLogFile` above...

Reply:
If it works like this, I'm fine with it.